src/Libraries/RevitNodes/Elements/AbstractFamilyInstance.cs

```diff
@@ -92,9 +92,9 @@ public Point Location
         public override string ToString()
         {
             if (InternalFamilyInstance != null && InternalFamilyInstance.IsValidObject)
-                return string.Format("Family={0}, Type={1}", InternalFamilyInstance.Symbol.Name, InternalFamilyInstance.Name);
+                return string.Format(Properties.Resources.Family + "={0}, " + Properties.Resources.Type + "={1}", InternalFamilyInstance.Symbol.Name, InternalFamilyInstance.Name);
```

Comment:
I worry a little about localizing the `toString()`'s of elements - this means that if someone's code is comparing toString() between revit languages - it will now fail, when it worked before...
⚠️

Reply:
maybe not important, but definitely should be considered.

Reply:
@Amoursol any idea if people do this?

Reply:
toString() is used to show the results of this node, as the picture showed
![image](https://user-images.githubusercontent.com/33445445/59397007-a2bad780-8dbd-11e9-94d2-e3e39c56894a.png)
I don't know if this needs improvement or not, what do you think of this?

Reply:
Yep, I understand that the `toString()`  is used to show the output in preview and watch - but I consider this a part of the evaluated code since I know that users sometimes serialize their data using `stringFromObject` and `stringFromArray`, and not a part of the UI - though really I guess it is both.

I believe this design was intentional - the same way that the node names themselves are not localized - only descriptions and tooltips are  - to facilitate data and code staying static between different languages. 

@kronz any thoughts about this?

That's my opinion any way 😄 but up to you guys of course.

Reply:
@ZiyunShang Thinking of the impact of all the `ToString()` changes as well as other changes in this PR. They will make a subset values in preview bubble localized comparing to the rest of the cases. This would make Dynamo node preview bubble working in a in consistent way, please consider testing your changes as a localized build in a localized Revit env and you should see users will get preview bubbles in mixed languages which might not be so great. I suggest we hold on committing to these as part of 2.2.1.

Reply:
@QilongTang Yep, it seems like this commit would have a big impact on Dynamo, not only **ToString()**, but also the **private const string** like https://github.com/DynamoDS/DynamoRevit/blob/8dd8ee352097b348a8df259ded648731dddf6077/src/Libraries/RevitNodesUI/Selection.cs#L607 , this const string would be used by class SelectionBase<TSelection, TResult> in DynamoCore.
I will talk to the team about this commit, and now it seems that it is not a good time to make this change.

Reply:
> @Amoursol any idea if people do this?

I've not personally used, nor been exposed to this way of working beyond one singular use case of a South Korean Revit file containing Korean characters. However, I've only worked in English speaking locations so it could very well be problematic for those working in a different language and collaborating globally, albeit these kind of collaborations typically set English as the working language.

We should be careful to make meaningful differences between Dynamo based on localization.